markdown/articles/2020/12/20/historyofdoomsday.md

```diff
@@ -0,0 +1,1030 @@
+list at the time being known as 'Iggy Pop'. Iggy was a {U}{B} storm list based around the engine of {{Ill-Gotten Gains}}
+coupled with rituals effects, other fast mana, Lion's Eye Diamonds and {{Tendrils of Agony}} with tools like
+Mystical Tutor and {{Intuition}} to find them. The deck was very proactive, to the point it didn't run much
+protection at all. Discard effects like Duress weren't viable as, due to the mirrored recursion effect of 
+Ill-Gotten Gains, discarding disruption was pointless as the opponent could just return it to their hand 
+during Ill-Gotten Gain's resolution. The mana base was also very clean so pilots did not tend to splash for cards like 
+{{Orim's Chant}} and, because of the use of Lion's Eye Diamonds (LEDs), countermagic was not viable either. As a 
+result it was very all-in as far as combo decks go. With the deck you could loop Ill-Gotten Gains and, with a 
+Dark Ritual and an LED, you could net mana each loop. To close the loop you could Intuition for 3 copies
+of Tendrils of Agony to win. It was very consistent boasting 8 maindeck tutor effects and maindeck removal like
+{{Chain of Vapor}} or {{Cabal Pit}}.
+
+It's here in our story where we re-focus on Emidln. Emidln had gotten a taste of competetive magic and so 
+started browsing forums to find out more information. Places like the Starcitygames forums, the mana drain
+forums (in the new legacy section, the mana leak) and the mtgsalvation forums. He started to meet and 
+become friends with people online, chatting via Aim, MSN, IRC, ICQ and a whole host of other messaging platforms.
+He realised he very much enjoyed the idea and style of the combo decks that were coming out and was invited
+to a specialised team forums for 'Team Blitzkrieg' or BZK for short. On the BZK forums all mana of combo decks
+were being discussed including early Spanish Inquisition lists (SI), the aforementioned Iggy Pop and some new
+decks that would start to evolve into more Modern storm lists and Emidln was in the middle of them.
+
+Spanish Inquistion is a deck that a lot of people still hold very dear to their heart to this day. The idea of 
+the deck is to utilise fast mana and ritual effects combined with 'draw 4's' like {{Cruel Bargain)) and
+{{Infernal Contract}}. {{Land Grant}}, along with {{Culling the Weak}} and creatures like {{Phyrexian Walker}}
+were used to cheat on mana and then the deck finished off opponents with either Tendrils of Agony or
+{{Goblin Charbelcher}}. There was a point in time where the SI decks tried a number of cards in them to 
+increase the *business* count available as the draw 4's could be unreliable. Cards like {{Diablic Intent}} worked
+alongside the 'tallmen' approach with {{Shield Sphere}} and friends however this did stretch reliance on the creature
+aspect. Doomsday was explored by Emidln and BZK friends; Evenpence and Desolutionist as a way to incorporate additional 
```

Comment:
I don't think I mentioned B.C., Blaine Christiansen with SI, but we was an early contributor to SI from the same forum as Evenpence and myself. He played and posted about the deck early on, having some success at midwestern events piloting SI, usually red-based for ETW.